app/Jobs/ValidateAndInstallServerJob.php

```
@@ -0,0 +1,162 @@
+            Log::error('ValidateAndInstallServer: Exception occurred', [
+                'server_id' => $this->server->id,
+                'error' => $e->getMessage(),
+                'trace' => $e->getTraceAsString(),
+            ]);
+
+            $this->server->update([
+                'validation_logs' => 'An error occurred during validation: '.$e->getMessage(),
+                'is_validating' => false,
+            ]);
+        }
+    }
```

PHPMD (2.15.0)

The method handle() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)